### ISSUE TITLE
PersonSelect - export the clear selection from controller

### ISSUE THREAD

**Issue by eikeland** [equinor/fusion-react-components#1316]
**Is your feature request related to a problem? Please describe.**
It would be nice to be able to clear the selection with a callable function.

**Describe the solution you'd like**
export new function from controller that clears any selection.

**Comment by AndreasPresthammer** [comment]
The lack of the ability to 'clear' out the value is blocking our current usage of the `<PersonSelect>` component.

Repro:
1. Have a component using it's own state to track the selected person.
`const [selectedUserId, setSelectedUserId] = useState<string>();
2. Set the state from the `PersonSelect` component using the `onSelect` callback.
```
<PersonSelect
  onSelect={(e: any) => {
    const azureUniqueId = e.nativeEvent?.detail?.selected?.azureUniqueId as string | undefined;
    setSelectedUserId(azureUniqueId);
  }}
/>
```
3. Clicking the 'Clear' will clear out the UI, but there does not seem to be (as far as I've found) any callback called upon clicking the 'Clear' button.
![image](https://github.com/equinor/fusion-react-components/assets/328539/4485c9f1-c8b4-4d54-9b22-a94e735b3956)

The result of the above is that the selector will be left in a 'cleared' state, but the state of `selectedUserId` will be the previously selected value. Storing a form based on this is unpredictable, and can lead users to register the wrong data.

**Comment by eikeland** [comment]
A fix for this is out today, @equinor/fusion-react-person@0.6.6.
Now the onSelect will fire when selecting and clearing/desecting person.

**Comment by AndreasPresthammer** [comment]
Just tested `@equinor/fusion-react-person@0.6.6` and it does _not_ seem to have fixed the problem.

Clicking the 'clear' (x) button does not invoke the onSelect handler.

**Comment by eikeland** [comment]
i will do some test on this and reach out if more info is needed.

**Comment by eikeland** [comment]
fusion-cli had a reference to older version of fusion-react-person,
Fixed in @equinor/fusion-cli@3.1.4

**Comment by eikeland** [comment]
Refernce: AB#48390
